.github/workflows/mirror-tooling-images.yml

```diff
@@ -0,0 +1,58 @@
+name: "[Daily] Mirror Tooling Images to GHCR"
+
+on:
+  schedule:
+    - cron: "0 3 * * *" # every day at 3 AM UTC
+  workflow_dispatch:
+    inputs:
+      ghcr_namespace:
+        description: "GHCR Namespace"
+        required: true
+        default: "ghcr.io/project-copacetic/copacetic"
+
+env:
+  GHCR_NAMESPACE: ${{ github.event.inputs.ghcr_namespace || 'ghcr.io/project-copacetic/copacetic' }}
+
+jobs:
+  mirror:
+    runs-on: ubuntu-latest
+    permissions:
+      packages: write
+      contents: read
+
+    strategy:
+      matrix:
+        images:
+          - "mirror.gcr.io/debian:10-slim"
+          - "mirror.gcr.io/debian:11-slim"
+          - "mirror.gcr.io/debian:12-slim"
+          - "mirror.gcr.io/debian:stable-slim"
+          - "mirror.gcr.io/ubuntu:20.04"
+          - "mirror.gcr.io/ubuntu:22.04"
+          - "mcr.microsoft.com/azurelinux/base/core:3.0"
+          - "mcr.microsoft.com/cbl-mariner/base/core:2.0"
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
```

Comment:
pin to digest and add comment with version, you can see other actions as an example

Reply:
done!